extension/src/views/gradleTasks/DependencyUtils.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from 'vscode';
+import { DependencyItem, GradleDependencyType } from '../../proto/gradle_pb';
+import { DependencyConfigurationTreeItem } from './DependencyConfigurationTreeItem';
+import { DependencyTreeItem } from './DependencyTreeItem';
+import { ProjectDependencyTreeItem } from './ProjectDependencyTreeItem';
+export const GRADLE_OMITTED_REVEAL = 'gradle.omitted.reveal';
+
+export function protocolItem2ProjectDependencyTreeItem(
+  protocolItem: DependencyItem,
+  parent: vscode.TreeItem
+): ProjectDependencyTreeItem | undefined {
+  const name = 'Dependencies';
+  const projectItem: ProjectDependencyTreeItem = new ProjectDependencyTreeItem(
+    name,
+    vscode.TreeItemCollapsibleState.Collapsed,
+    parent
+  );
+  const children = protocolItem.getChildrenList();
+  const treeChildren = [];
+  for (const child of children) {
+    if (child.getType() !== GradleDependencyType.CONFIGURATION) {
+      continue;
+    }
+    const configurationItem = protocolItem2DependencyConfigurationTreeItem(
+      child,
+      projectItem
+    );
+    if (configurationItem) {
+      treeChildren.push(configurationItem);
+    }
+  }
+  if (!treeChildren.length) {
+    return undefined;
+  }
+  projectItem.setChildren(treeChildren);
+  return projectItem;
+}
+
+export function protocolItem2DependencyConfigurationTreeItem(
```

Comment:
why export? I don't find any other caller.

Reply:
Will remove this redundant `export`s.